andela/hc-beef-sauce
# #158532837 user check assigning

#### What does this PR do?
Assign specific checks to a user when inviting them to join your team.
#### Description of Task to be completed?
The user is expected to be able to assign specified checks to another user when inviting them to join their team.
#### How should this be manually tested?
To test it, load the user settings page and click on the Invite a Team Member button.
A modal form opens up in which the user can input the email of the person to be invited along with the specific checks to be assigned. 
Once submitted(Send Invite button clicked), the invited user should be able to see assigned checks on their dashboard.
What are the relevant pivotal tracker stories?
#158532837
#### Screenshots
<img width="1440" alt="screen shot 2018-07-06 at 08 18 58" src="https://user-images.githubusercontent.com/37480490/42361103-5672081a-80f5-11e8-9e81-c30e34ce797d.png">

<img width="1440" alt="screen shot 2018-07-07 at 20 40 56" src="https://user-images.githubusercontent.com/37480490/42418765-b97006a4-82af-11e8-9aa2-309e69c878ba.png">

<img width="1321" alt="screen shot 2018-07-08 at 12 58 19" src="https://user-images.githubusercontent.com/37480490/42418770-ced5aa9e-82af-11e8-84bc-1bd6e38758e6.png">

<img width="1359" alt="screen shot 2018-07-08 at 12 58 37" src="https://user-images.githubusercontent.com/37480490/42418771-db7f2e64-82af-11e8-9b68-05d398dec3e3.png">
